pkg/cli/cli.go

```diff
@@ -218,22 +216,32 @@ func (c *cli) initialize() error {
 	// layout and --plugins values can be short (ex. "go/v2.0.0") or unversioned
```

Comment:
```suggestion
	// layout and --plugins values can be short (ex. "go/v2.0") or unversioned
```

Reply:
We haven't yet removed patch versions yet, PR to come.